.sync.yml

```diff
@@ -31,20 +31,6 @@ appveyor.yml:
     CHECK: parallel_spec
   simplecov: true
 Gemfile:
-  required:
-    ":system_tests":
-    - gem: puppet-module-posix-system-r#{minor_version}
```

Comment:
More of a query because I was working with this gem recently - should they not be in all modules? I'm guessing the PDK removed this?

Reply:
It's still included in :development. :system_tests is just no longer a thing with litmus